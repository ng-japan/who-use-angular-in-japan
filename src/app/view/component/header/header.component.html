<header fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="32px">
  <div class="mat-display-1">Who use Angular in Japan?</div>

  <div class="mat-body-1">
    日本国内でAngular/AngularJSを利用している会社・組織をリストアップしています。<br>
    The list of companies who use Angular/AngularJS in Japan.
  </div>

  <a mat-stroked-button class="add-organization-button" href="https://github.com/ng-japan/who-use-angular-in-japan/blob/master/ADD_ORGANIZATION.md">
    <div fxFlex="row" fxLayoutAlign="center center" fxLayoutGap="8px">
      <div>Add your Organization</div>
      <mat-icon>add</mat-icon>
    </div>
  </a>
</header>
