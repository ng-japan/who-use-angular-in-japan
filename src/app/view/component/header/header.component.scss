@import '../../../../styles/theme';

:host {
  display: block;
}

header {
  color: mat-contrast($app-primary, 500);
  background: mat-color($app-primary);
  padding: 48px 32px;
}

[mat-stroked-button].add-organization-button  {
  border-color: #ffffff;
}
