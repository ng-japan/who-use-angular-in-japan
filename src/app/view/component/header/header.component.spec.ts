import { NO_ERRORS_SCHEMA } from '@angular/core';
import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
import { HeaderComponent } from './header.component';

describe('HeaderComponent', () => {
  let component: HeaderComponent;
  let fixture: ComponentFixture<HeaderComponent>;

  beforeEach(waitForAsync(() => {
    TestBed.configureTestingModule({
      declarations: [HeaderComponent],
      schemas: [NO_ERRORS_SCHEMA],
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(HeaderComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  test('should create', () => {
    expect(component).toBeTruthy();
  });

  test('matching snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
