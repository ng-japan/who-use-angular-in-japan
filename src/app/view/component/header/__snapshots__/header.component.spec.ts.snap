// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent matching snapshot 1`] = `
<app-header>
  <header
    fxlayout="column"
    fxlayoutalign="center center"
    fxlayoutgap="32px"
  >
    <div
      class="mat-display-1"
    >
      Who use Angular in Japan?
    </div>
    <div
      class="mat-body-1"
    >
       日本国内でAngular/AngularJSを利用している会社・組織をリストアップしています。
      <br />
       The list of companies who use Angular/AngularJS in Japan. 
    </div>
    <a
      class="add-organization-button"
      href="https://github.com/ng-japan/who-use-angular-in-japan/blob/master/ADD_ORGANIZATION.md"
      mat-stroked-button=""
    >
      <div
        fxflex="row"
        fxlayoutalign="center center"
        fxlayoutgap="8px"
      >
        <div>
          Add your Organization
        </div>
        <mat-icon>
          add
        </mat-icon>
      </div>
    </a>
  </header>
</app-header>
`;
